src/scrabble.js

```diff
@@ -1,5 +1,44 @@
-function scrabble() {
+const {
+  stringIsValid,
+  findDoubleChars,
+  findTripleChars,
+  getScoreOfPlainString,
+  outermostBracketsAreDouble,
+  outermostBracketsAreTriple
+} = require('./scripts.js')
```

Comment:
Nice work separating out functions to a different file, keeps everything nice and tidy :)